packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -243,6 +243,20 @@ abstract class WidgetsBindingObserver {
   void didChangeAccessibilityFeatures() {}
 }
 
+/// An abstract class for service extension registration.
+///
+/// As BindingBase is a singleton in Flutter framework, and runApp will always ensure there will be
```

Comment:
Don't explain the history of why this is created. Just explain what this does and how to use it.